x-pack/plugins/infra/public/pages/metrics/hosts/hooks/use_host_table.test.ts

```diff
@@ -34,7 +34,7 @@ describe('useHostTable hook', () => {
             avg: 34359.738368,
           },
         ],
-        path: [{ value: 'host-0', label: 'host-0', os: null }],
+        path: [{ value: 'host-0', label: 'host-0', os: null, cloudProvider: 'aws' }],
```

Comment:
Not related to this change, but could we declare `nodes` as `nodes: SnapshotNode[] = {...`? This will make it easier for us to make sure we're building the right object. Apparently if you pass an invalid attribute in `path`, the code will not complain.